core/btdiscovery.cpp

```diff
@@ -31,8 +31,8 @@ static dc_descriptor_t *getDeviceType(QString btName)
 		else if (btName.mid(4,2) == "2-") product = "OSTC 2N";
 		else if (btName.mid(4,2) == "+ ") product = "OSTC 2";
 		// all OSTCs are HW_FAMILY_OSTC_3, so when we do not know,
```

Comment:
This comment isn't that correct. Not all OSTC's are HW_FAMILY_OSTC_3, but, rather, all with bluetooth are HW_FAMILY_OSTC_3.

Reply:
But, there's also the Frog, which is a HW computer with bluetooth, which isn't a HW_FAMILY_OSTC_3, but there weren't that many of them made, and I've actually never seen one, so lets ignore that =)

Reply:
I had one conversation with a Frog owner a few years ago... but yes, they are being ignored here :-)